lib/gollum/public/gollum/stylesheets/wiki_content.scss

```diff
@@ -0,0 +1,157 @@
+// Our own styling for everything in the page and subpage body.
+// Should be included *after* primer styling to override the latter.
+
+@import '_base', 'highlights.scss';
+
+.markdown-body {
+    padding: 1em 1em 1em 0;
+    font-size: 15px;
+    line-height: 1.7;
+    word-wrap: break-word;
+
+    img {
+        max-width: 100%;
+    }
+
+    table {
+      border-collapse: collapse;
+      border: $border-standard;
+      border-radius: 3px !important;
+      
+      tr {
+        border-top: 0;
+      }
+      tr th {
+        border-top: 0;
+      }
+      th:first-child {
+        border-left: 0;
+      }
+      th:last-child {
+        border-right: 0;
+      }
+      tr:first-child td {
+        border-top: 0;
+      }
+      tr td:first-child {
+        border-left: 0;
+      }
+      tr:last-child td {
+        border-bottom: 0;
+      }
+      tr td:last-child {
+        border-right: 0;
+      }
+      tfoot {
+        border-top: $border-standard;
+        th {
+          border-bottom: 0;
+        }
+      }
+    }
+    
+    code, pre, tt {
+      font-family: $font-console;
+      font-size: 12px;
+    }  
+
+    pre {
+        border: 1px solid #ccc;
+        overflow: auto;
+        border-radius: 3px;
+        padding-bottom: 8px;
+        padding-top: 8px;
+        position: relative;
+
+        /* Prevent the overflow of copy to clipboard button */
+        min-height: 3rem;
+
+        > tt, > code {
+            margin: 0;
+            padding: 0;
+            white-space: pre;
+            border: none;
+            background: transparent;
+        }
+
+        button.btn-octicon {
```

Comment:
@benjaminwil since we're not using the Primer [marketing component](https://primer.style/css/marketing/) I had to style the copy-to-clipboard button here. Does that interfere with the work you're doing that you just mentioned in #1942? If so I can remove this, or even add the marketing component back in, if you feel that would be useful

Reply:
I don't think it interferes!

Reply:
I really would favor a solution though that uses Primer utilities, rather than styling this button ourselves. 

[Here](https://primer.style/css/utilities/marketing-layout) it is explained that the marketing layout utilities are built on top of the [Primer core utilities](https://primer.style/css/utilities/layout#position) so presumably we could position the button using only core utility classes?

Reply:
That would be my preference to, and absolutely how I will approach refactoring this button in #1901.

Reply:
As a heads up, it seems that (very surprisingly) only `right-0` (and not `right-1` etc.) is defined in the core utilities. That's what led me to style the button myself. Or maybe I overlooked something.

Anyway, I'll merge this PR with the self-styled button, since @benjaminwil will take care of it in #1901! Many thanks.